sample/src/androidTest/java/com/schibsted/spain/barista/sample/MenuClickTest.java

```diff
@@ -46,18 +47,21 @@ public void iconMenuClick_byTitle() {
   }
 
   @Test
+  @Ignore
```

Comment:
Those are flaky tests on CI, we must try to check why...

Reply:
I think they pass in the local machine...

Reply:
Yes, that's why I marked them as flaky, maybe we can merge this PR and try to fix the tests?

Reply:
You marked them as `@Ignore`, not flaky.

Perwaps we could add a message that explains that it doesn't pass on CI but it does in the local machine? Just to help to ourselves of the future.

Reply:
Let's see if `@FlakyTest` works

Reply:
Why do they fail in CI? Do we have any idea?
Maybe we could try running it locally with the exact same emulator as CI, sometimes that gives it away 🤷‍♂